webrender/src/device/gfx/device.rs

```diff
@@ -461,6 +493,16 @@ impl<B: hal::Backend> Device<B> {
             (limits.non_coherent_atom_size - 1) as usize,
         );
 
+        let quad_buffer = VertexBufferHandler::new(
```

Comment:
note: we don't *actually* need this buffer. May as well just hard-code the positions in the vertex shaders.
That is to say, it's fine to let it stay. Just saying there is another option :)

Reply:
@kvark are you refering to store the quad data in an array in the VS and use `gl_VertexID` to get the index for the current vertex?

Reply:
https://github.com/kvark/vange-rs/blob/8ba08f72728afcdcb2c6e1100180d5f1d101b52a/data/shader/downsample.glsl#L13-L21